GTG/core/config.py

```diff
@@ -180,10 +180,14 @@ def save(self):
         self._save_function()
 
 
-class CoreConfig():
+class CoreConfig_():
```

Comment:
Why the underscore here?

Reply:
For backwards compatibility: `CoreConfig` is redefined below:

```
def CoreConfig():
    return CoreConfig_.get_instance()
```